test/specs/billing.js

```diff
@@ -56,6 +56,16 @@ describe.only('/billing', () => {
       });
   });
 
+  it('returns a PEL fee of zero if the establishment was inactive', () => {
+    return request(this.api)
+      .get('/establishment/101/billing?year=2018')
+      .expect(200)
+      .expect(response => {
+        assert.equal(response.body.data.pel, 0);
+        assert.equal(response.body.data.total, response.body.data.pils);
+      });
+  });
+
```

Comment:
can we get a test for the revocation date as well please?

Reply:
Yes.